frontend/config-assistant/src/components/toolbar/TopMenuBar.ts

```diff
@@ -231,7 +228,6 @@ export class TopMenuBar {
         command: () => {
           this.sessionStore.currentEditorWrapper.undo();
         },
-        disabled: () => !useSessionStore().currentEditorWrapper.hasUndo(),
```

Comment:
why do you remove this here everywhere?

Reply:
because this can't rightly judge the state of hasUndo or hasRedo.
Even it is able to undo, button is still disabled.

Reply:
Are you sure? Because last time it worked

Reply:
If it doesn't work, then we can merge it like this, but then please create a new issue to make it work :)

Reply:
Still not working on the current state of this branch, I think I will create another issue to solve this. 
Btw, why is the first disable command different than others:
```
disabled: () => !storeToRefs(useSessionStore()).currentEditorWrapper.value.hasUndo(),
```
rather than
```
disabled: () => !useSessionStore().currentEditorWrapper.hasUndo(),
```